requirements.txt

```diff
@@ -10,4 +10,5 @@ gdown>=4.4.0
 grad-cam
 tabulate
 segmentation-models-pytorch
-opencv-python-headless==4.2.0.32 
\ No newline at end of file
+opencv-python-headless==4.2.0.32 
+wandb
```

Comment:
I intentionally don't want to include `wandb` inside installation. If the user want to use wandb logger, they must have it installed on their own.